Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/Dictionary.java

```diff
@@ -0,0 +1,7 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+public interface Dictionary {
+	
+	public String getRandomWord();
```

Comment:
Hmm, could you provide a link, I haven't heard of a case where interface methods aren't implicitly `public`.  I thought interfaces were about methods accessible outside of the class, i.e. public ones.